apps/src/aiDifferentiation/PersonalizationAnswers.tsx

```diff
@@ -0,0 +1,329 @@
+
+import style from './personalization-information.module.scss';
+
+interface NumberOfYearsTeachingAnswerProps {
+  yearsTeaching: number;
+  setYearsTeaching: (years: number) => void;
+}
+
+export const NumberOfYearsTeachingAnswer: React.FC<
+  NumberOfYearsTeachingAnswerProps
+> = ({yearsTeaching, setYearsTeaching}) => {
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const value = parseInt(event.target.value) || 0;
+    setYearsTeaching(value);
+  };
+
+  return (
+    <div className={style.numberOfYearsArea}>
+      <BodyOneText className={style.bodyText}>
+        I've been teaching for
+      </BodyOneText>
+      <input
+        type="number"
+        name="years-teaching"
+        className={style.answer}
+        value={yearsTeaching > 0 ? yearsTeaching : ''}
+        onChange={handleChange}
+      />
+      <BodyOneText className={style.bodyText}>years!</BodyOneText>
+    </div>
+  );
+};
+
+interface ClassroomVisionAnswerProps {
+  classroomVision: string;
+  setClassroomVision: (vision: string) => void;
+}
+
+export const ClassroomVisionAnswer: React.FC<ClassroomVisionAnswerProps> = ({
+  classroomVision,
+  setClassroomVision,
+}) => {
+  const handleChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setClassroomVision(event.target.value);
+  };
+
+  return (
+    <div>
+      <StrongText>Describe your ideal classroom environment:</StrongText>
+      <textarea
+        name="classroom-vision"
+        value={classroomVision}
+        onChange={handleChange}
+        className={style.freeResponseBox}
+        rows={4}
+      />
+      For example, you could say, "A collaborative learning community" or "A
+      creative, engaging space"
+    </div>
+  );
+};
+
+interface ChallengeAnswerProps {
+  challenge: string;
+  setChallenge: (challenge: string) => void;
+}
+
+export const ChallengeAnswer: React.FC<ChallengeAnswerProps> = ({
+  challenge,
+  setChallenge,
+}) => {
+  const handleChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setChallenge(event.target.value);
+  };
+
+  return (
+    <div>
+      <StrongText>Describe your challenge</StrongText>
+      <textarea
+        name="classroom-challenge"
+        value={challenge}
+        onChange={handleChange}
+        className={style.freeResponseBox}
+        rows={4}
+      />
+    </div>
+  );
+};
+
+interface ConfidenceAnswerProps {
+  selectedConfidence: number;
+  setSelectedConfidence: (confidence: number) => void;
+}
+
+export const ConfidenceAnswer: React.FC<ConfidenceAnswerProps> = ({
+  selectedConfidence,
+  setSelectedConfidence,
+}) => {
+  const handleButtonClick = (index: number) => {
+    setSelectedConfidence(index);
+  };
+
+  const determineColor = (index: number) => {
+    if (selectedConfidence === index) {
+      return buttonColors.purple;
+    } else {
+      return buttonColors.white;
+    }
+  };
+
+  return (
+    <div>
+      <div className={style.confidenceButtons}>
```

Comment:
I'm a little surprised you have to make all of these buttons individually; do we not have a likert scale picker component anywhere? It's not a big deal, just maybe something to flag with DSCO or MUI or whoever is thinking about reusable components at this point 😅

Reply:
Yeah, I also thought it was weird.  I checked with Molly on the ["Chip" vs. button](https://codedotorg.slack.com/archives/C050HE4QDHR/p1757614637729189), but I will also ask in the design group about this.